Git Conventions

```diff
@@ -0,0 +1,13 @@
+Git Conventions
```

Comment:
I think this file devia estar na organization

Reply:
yeah, u guys can organize it convetionally